crates/ln-dlc-node/src/ldk_node_wallet.rs

```diff
@@ -40,6 +41,7 @@ where
     settings: RwLock<WalletSettings>,
     fee_rate_estimator: Arc<F>,
     locked_outpoints: Mutex<Vec<OutPoint>>,
+    node_storage: Arc<dyn Storage + Send + Sync + 'static>,
```

Comment:
can this be done with static dispatch? We probably lose a bit performance here, but I guess we're not using that too often for us to matter.

Reply:
I just followed the same pattern we were using in `ln_dlc_wallet`. But that doesn't mean it's necessarily good!

You are right that we could use static dispatch, but then we have to propagate the type parameter and I will be unhappy. Ideally we get rid of the ill-conceived `Storage` trait and then we don't have to worry about it anymore :D